src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -318,22 +318,10 @@ namespace AppInstaller::CLI::Workflow
         }
     }
 
-    void ExecuteInstaller(Execution::Context& context)
+    void ExecuteInstallerForType(Execution::Context& context, InstallerTypeEnum installerType)

```

Comment:
The intention was to create a `WorkflowTask` derivative.  Example:
https://github.com/microsoft/winget-cli/blob/18ee6999f442d1b9f6015cefc4ad22804e744978/src/AppInstallerCLICore/Workflows/InstallFlow.h#L42

That allows the continued use of the stream operators rather than checking `IsTerminated`.

Reply:
Created a `WorkflowTask` derivative for ExecuteInstallerForType